modyn/tests/supervisor/internal/triggers/test_datadrifttrigger.py

```diff
@@ -13,13 +13,13 @@
     AlibiDetectMmdDriftMetric,
 )
 from modyn.config.schema.pipeline.trigger.drift.config import AmountWindowingStrategy
-from modyn.config.schema.pipeline.trigger.drift.detection_window import (
-    TimeWindowingStrategy,
-)
-from modyn.config.schema.pipeline.trigger.drift.metric import (
```

Comment:
i feel like your ruff or whatever has a stricter line limit than the repo because this keeps changing despite the CI running through :D

Reply:
That's right; it also changes a couple of other things on auto-save, but those will be reverted by the pre-commit hooks. The only thing that stays in the shorter lines iff a trailing comma is used. So far, it hasn't bothered me enough to investigate it. All my other code bases have shorter line length settings😅